WeatherMonitoringAndReportingService.BL/Utilities/ReadWeatherDataUtility.cs

```diff
@@ -0,0 +1,21 @@
+namespace WeatherMonitoringAndReportingService.BL.Utilities;
+
+public static class ReadWeatherDataUtility
+{
+    public static string? GetJsonWeatherData()
+    {
+        Console.WriteLine("Please add the JSON data you want ot add!");
+
+        var jsonData = Console.ReadLine();
+
+        return jsonData;
```

Comment:
please dont add variables that short living without any responsibility 
keep your code tight and clean